Classes/TMRequestFactory.m

```diff
@@ -64,7 +64,7 @@ - (NSURL *)baseURL {
         parameters[@"before_timestamp"] = @(round([beforeDate timeIntervalSince1970]));
     }
 
-    return [self requestWithPath:[[NSString alloc] initWithFormat:@"blog/%@/notes", blogUUID] method:TMHTTPRequestMethodGET queryParameters:[parameters copy]];
+    return [self requestWithPath:[[NSString alloc] initWithFormat:@"blog/%@/notes", TMURLEncode(blogUUID)] method:TMHTTPRequestMethodGET queryParameters:[parameters copy]];
```

Comment:
are we sure `TMURLEncode` encodes this the correct way?

Reply:
I've tested that the encoding it creates works against the api. Is that what you're thinking of?

Reply:
👍